extensions/interactions/GraphInput/protractor.js

```diff
@@ -18,7 +18,7 @@
 
 var forms = require('../../../core/tests/protractor_utils/forms.js');
 var objects = require('../../objects/protractor.js');
-var until = protractor.ExpectedConditions;
+var waitFor = require('../../,,/core/tests/protractor_utils/waitFor.js');
 
```

Comment:
Change this `/,,/` to `/../`. Also I should think you should run at least 1 describe spec from editorAndPlayer just to make sure the imports are set correctly.

Reply:
This is for graphInput so just run extensions 's describe "All interaction" spec

Reply:
Yep, sorry, you're absolutely right. I'd usually do that, but was in a rush this morning, so apologies for that.

Will work on this (and the other comments) when I get back and wait for Travis to pass before pinging again. Thanks for taking a look!

Reply:
Done!